cereal/log.capnp

```diff
@@ -2246,6 +2246,7 @@ struct LiveParametersData {
   angleOffsetAverageValid @18 :Bool = true;
   steerRatioValid @19 :Bool = true;
   stiffnessFactorValid @20 :Bool = true;
+  calibrationValid @21 :Bool;
```

Comment:
why not just check liveCalibration? duplicate field

Reply:
since calibration now affects .valid, there may be some false-positives if selfdrived processes liveCalibration before paramsd does